packages/integration/lib/src/views/poll/create_poll_screen.dart

```diff
@@ -0,0 +1,813 @@
+  LMFeedWidgetUtility _widgetsBuilder = LMFeedCore.widgetUtility;
+  LMUserViewData? user = LMFeedLocalPreference.instance.fetchUserData();
+  List<String> options = ["", ""];
+  final ValueNotifier<bool> _optionBuilder = ValueNotifier(false);
+  String exactlyDialogKey = 'exactly';
+  int exactlyValue = 1;
+  ValueNotifier<bool> _advancedBuilder = ValueNotifier(false);
+  final TextEditingController _questionController = TextEditingController();
+  ValueNotifier<DateTime?> _expiryDateBuilder = ValueNotifier(null);
+  ValueNotifier<PollMultiSelectState> _multiSelectStateBuilder =
+      ValueNotifier(PollMultiSelectState.exactly);
+  ValueNotifier<bool> _rebuildMultiSelectStateBuilder = ValueNotifier(false);
+  ValueNotifier<int> _multiSelectNoBuilder = ValueNotifier(1);
+  ValueNotifier<bool> _pollTypeBuilder = ValueNotifier(false);
+  ValueNotifier<bool> _isAnonymousBuilder = ValueNotifier(false);
+  ValueNotifier<bool> _allowAddOptionBuilder = ValueNotifier(false);
+
+  Future<DateTime?> showDateTimePicker({
+    required BuildContext context,
+    DateTime? initialDate,
+    DateTime? firstDate,
+    DateTime? lastDate,
+  }) async {
+    initialDate ??= DateTime.now();
+    firstDate ??= DateTime.now();
+    lastDate ??= firstDate.add(const Duration(days: 365 * 200));
+
+    final DateTime? selectedDate = await showDatePicker(
+        context: context,
+        initialDate: initialDate,
+        firstDate: firstDate,
+        lastDate: lastDate,
+        helpText: 'Start date',
+        builder: (context, child) {
+          return Theme(
+              data: Theme.of(context).copyWith(
+                colorScheme: ColorScheme.light(
+                  primary: theme.primaryColor,
+                  onPrimary: theme.onPrimary,
+                ),
+              ),
+              child: child!);
+        });
+
+    if (selectedDate == null) return null;
+
+    if (!context.mounted) return selectedDate;
+
+    final TimeOfDay? selectedTime = await showTimePicker(
+      context: context,
+      initialTime: TimeOfDay.fromDateTime(initialDate),
+      builder: (context, child) {
+        return Theme(
+            data: Theme.of(context).copyWith(
+              colorScheme: ColorScheme.light(
+                primary: theme.primaryColor,
+                onPrimary: theme.onPrimary,
+              ),
+            ),
+            child: child!);
+      },
+    );
+    if (selectedTime == null) return null;
+    DateTime selectedDateTime = DateTime(
+      selectedDate.year,
+      selectedDate.month,
+      selectedDate.day,
+      selectedTime.hour,
+      selectedTime.minute,
+    );
+
+    if (selectedDateTime.isBefore(DateTime.now())) {
+      showSnackBar('Expiry date cannot be in the past');
+      return null;
+    }
+
+    return selectedDateTime;
+  }
+
+  String getFormattedDate(DateTime date) {
+    String day = date.day < 10 ? '0${date.day}' : '${date.day}';
+    String month = date.month < 10 ? '0${date.month}' : '${date.month}';
```

Comment:
We can use `padLeft(2, '0');` for adding padding to the left

Reply:
done